docs/journey/day1/README.md

```diff
@@ -369,7 +369,7 @@ being executed.
 
 ![PipelineRun doing a dry run of the configuration](img/pipelinerun-dryrun.png)
 
-This validates that the YAML can be applied, by executing a `kubectl apply --dry-run`.
+This validates that the YAML can be applied, by executing a `oc apply -k --dry-run`.
```

Comment:
We're using both `oc` and `kubectl` in different places. Maybe we can agree on one tool and standardize in a different PR